README.md

```diff
@@ -1,7 +1,8 @@
-
 <h3 align="center">Homarr</h3>
   <p align="center">
-<img align="end" width=500 src="https://user-images.githubusercontent.com/49837342/168315259-b778c816-10fe-44db-bd25-3eea6f31b233.png" />
+  <a href="">
```

Comment:
Why are you adding an href here??

Reply:
@walkxcode oh was testing and forgot to remove that part. If you click on image, it will take you to a different location (like youtube.com/video-id) then image file location. It doesn't work right now tho, it just links to a broken link

can see and test here https://github.com/c00ldude1oo/homarr#readme